### PR TITLE
[#1701] fix(mysql): Compatible with lower versions of MySQL driver, unable to obtain table comments.

### PR DESCRIPTION
**Pull request by Clearvive**
### What changes were proposed in this pull request?
Compatible with lower versions of MySQL driver, unable to obtain table comments.

### Why are the changes needed?
Fix: #1701

### Does this PR introduce _any_ user-facing change?
NO

### How was this patch tested?
UT/IT

### CODE REVIEW

File: catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java [hunk 1]
```diff
@@ -49,6 +50,9 @@ public JdbcTable load(String databaseName, String tableName) throws NoSuchTableE
             String.format("Table %s does not exist in %s.", tableName, databaseName));
       }
       String comment = table.getString("REMARKS");
+      if (StringUtils.isEmpty(comment)) {
+        comment = loadCommentFromSysTable(connection, tableName);
```
Comment (mchades) [inline catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java:54]: Do we need to provide a unified solution at the JDBC catalog level for handling missing values in the Driver?

Comment (mchades) [reply catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java:54]: The implementations can be different, but do they need to implement a unified interface?

File: catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java [hunk 2]
```diff
@@ -49,6 +50,12 @@ public JdbcTable load(String databaseName, String tableName) throws NoSuchTableE
             String.format("Table %s does not exist in %s.", tableName, databaseName));
       }
       String comment = table.getString("REMARKS");
+      if (StringUtils.isEmpty(comment)) {
+        // In Mysql version 5.7, the comment field value cannot be obtained in the driver API.
+        LOG.warn(
+            "Not found comment in mysql driver api. Will try to get comment from the system table");
+        comment = loadCommentFromSysTable(connection, tableName);
```
Comment (FANNG1) [inline catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java:57]: is this problem still exits in the newest version?

**Review by yuqi1129 — APPROVED** [review]
It's Okay with me.
@jerryshao Please help to take a look, This PR will block all MRs.

**Review by FANNG1 — DISMISSED** [review]

File: catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java [hunk 3]
```diff
@@ -49,6 +50,12 @@ public JdbcTable load(String databaseName, String tableName) throws NoSuchTableE
             String.format("Table %s does not exist in %s.", tableName, databaseName));
       }
       String comment = table.getString("REMARKS");
+      if (StringUtils.isEmpty(comment)) {
+        // In Mysql version 5.7, the comment field value cannot be obtained in the driver API.
+        LOG.warn(
+            "Not found comment in mysql driver api. Will try to get comment from the system table");
```
Comment (jerryshao) [inline catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java:56]: Actually you didn't get "comment" from system table, so you'd better update the log here and method name below.

**Review by jerryshao — APPROVED** [review]

**Pull request merged**